### DISCUSSION TITLE
Error while moving plot from temp storage to NFS share

### DISCUSSION THREAD

**Discussion by Rukahs** [Chia-Network/chia-blockchain#5313] [Support]
When moving files from local storage to NFS share permissions cannot preserved and throws an error, but the move completes successfully. NFS client is linux and NFS server is FreeBSD. client can read/write correctly to the NFS share.

The chia plotter sees the error and throws error message saying cannot copy, retrying in 5 minutes. Never finishing the plot.

python open() and file.write() commands work correctly. 
shutil.move() throws an exception and does not move the file. (I think this is the problem)
shutil.copy() does move (copy) the file successfully with no errors.

**Comment by JuEnPeHa** [comment]
Its lost, It happen to me, plots lost if the connection fail for a little bit.

**Comment by Rukahs** [comment]
Connection to NFS share is never interrupted, and connection to the TMP storage is never interrupted. 
Plotter just errors (i think on the inability to preserve file permissions) and re-tries in 5 minutes... continuously.

I skimmed through create_plots.py but didn't see where it does the final move from tmp storage to destination.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
